wordle-web/src/scripts/letter.ts

```diff
@@ -0,0 +1,31 @@
+// Class that represents a letter
+
+export enum LetterStatus {
+    NotGuessed = 0,
+    Correct,
+    Misplaced,
+    Wrong
+}
+
+export class Letter {
+    letter: string;
+    status: LetterStatus = LetterStatus.NotGuessed;
+
+    constructor(letter: string) {
+        this.letter = letter;
+    }
+
+    get color() {
+        switch (this.status) {
+            case LetterStatus.NotGuessed:
```

Comment:
I think if you look at their code, they have the grey color setting listed last and as default. It probably doesn't matter too much either way tho.